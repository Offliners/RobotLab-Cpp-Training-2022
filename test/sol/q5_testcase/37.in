4-4-16*19*10/4=
