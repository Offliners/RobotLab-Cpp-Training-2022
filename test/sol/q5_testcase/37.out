1
-55
-84
-90
